apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -316,6 +317,68 @@ public void loadSampleUUnchanged() {
       if (entityManager != null) entityManager.close();
     }
   }
+
+  @Test
+  public void loadInitialEnrollmentShouldCount12() {
+    DataSource dataSource = DatabaseTestHelper.getTestDatabaseAfterClean();
+    loadSample(dataSource, Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
+
+    LoadAppOptions options = RifLoaderTestUtils.getLoadOptions(dataSource);
+    EntityManagerFactory entityManagerFactory =
+        RifLoaderTestUtils.createEntityManagerFactory(options);
+    EntityManager entityManager = null;
+    try {
+      entityManager = entityManagerFactory.createEntityManager();
+      Beneficiary beneficiaryFromDb = entityManager.find(Beneficiary.class, "567834");
+      Assert.assertEquals(12, beneficiaryFromDb.getEnrollments().size());
+
+    } finally {
+      if (entityManager != null) entityManager.close();
+    }
+  }
+
+  @Test
+  public void loadInitialEnrollmentShouldCount24() {
+    DataSource dataSource = DatabaseTestHelper.getTestDatabaseAfterClean();
+    loadSample(dataSource, Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
+    loadSample(dataSource, Arrays.asList(StaticRifResourceGroup.SAMPLE_U.getResources()));
+
+    LoadAppOptions options = RifLoaderTestUtils.getLoadOptions(dataSource);
+    EntityManagerFactory entityManagerFactory =
+        RifLoaderTestUtils.createEntityManagerFactory(options);
+    EntityManager entityManager = null;
+    try {
+      entityManager = entityManagerFactory.createEntityManager();
+
+      Beneficiary beneficiaryFromDb = entityManager.find(Beneficiary.class, "567834");
+      Assert.assertEquals(24, beneficiaryFromDb.getEnrollments().size());
+    } finally {
+      if (entityManager != null) entityManager.close();
+    }
+  }
+
+  @Test
+  public void loadInitialEnrollmentShouldCount20SinceThereIsAUpdateOf8Months() {
```

Comment:
I can't quite work out in my head if your logic above will correctly handle adding a month or two of data to the current year correctly...

You don't really have test coverage for that, so I'd suggest adding it. Maybe just as another step in this test case: what happens when one of the fields moves up to having ten months of data?

Reply:
talked about if it makes sense to leave it this way with karl

Reply:
I think I said something like, "if I were you and could reason my way to having a lot of confidence in the above logic, I _might_ skip adding the test -- otherwise, I'd double-check myself by adding a test case for it: what happens when we go from having just monthly data for 2020-01 to also having the data for 2020-02?"

Reply:
added an extra test for august to sept and how the file in sept could update fields in august.